<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="ip-172-31-26-20" ignored="0" name="Test" tests="3" failures="1" timestamp="23 Apr 2023 06:05:05 GMT" time="13.065" errors="0">
  <testcase name="test1" time="3.43" classname="testcases.LoginTest"/>
  <testcase name="test2" time="4.185" classname="testcases.LoginTest"/>
  <testcase name="test4" time="0.081" classname="testcases.LoginTest">
    <failure type="com.codoid.products.exception.FilloException" message="Invalid Query - SELECT * FROM DATA WHERE TESTNAME = TEST4&amp;apos;">
      <![CDATA[com.codoid.products.exception.FilloException: Invalid Query - SELECT * FROM DATA WHERE TESTNAME = TEST4'
at com.codoid.products.parser.SelectQueryParser.initializeQuery(SelectQueryParser.java:46)
at com.codoid.products.fillo.Select.getRecordset(Select.java:44)
at com.codoid.products.fillo.Connection.executeQuery(Connection.java:64)
at testcases.LoginTest.test4(LoginTest.java:51)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test4 -->
</testsuite> <!-- Test -->
